app/src/main/java/org/fossasia/openevent/general/ticket/Ticket.kt

```diff
@@ -23,9 +31,13 @@ data class Ticket(
         val position: String?,
         val quantity: String?,
 
-        val isHidden: Boolean? = false,
+        val isHidden: Boolean?,
         val salesStartsAt: String?,
         val salesEndsAt: String?,
-        val minOrder: String?
+        val minOrder: String?,
+        @ColumnInfo(index = true)
+        @Relationship("event")
+        var event: EventId?
```

Comment:
Is this statement working without initializing to null?

Reply:
no it should not , will change this